src/main/java/org/opentripplanner/graph_builder/module/configure/GraphBuilderModules.java

```diff
@@ -272,10 +274,32 @@ static DataImportIssueReporter provideDataImportIssuesToHTML(
   }
 
   @Provides
+  @Singleton
   static DataImportIssueSummary providesDataImportIssueSummary(DataImportIssueStore issueStore) {
     return new DataImportIssueSummary(issueStore.listIssues());
   }
 
+  @Provides
+  @Singleton
+  @Nullable
+  static StopConsolidationModule providesStopConsolidationModule(
+    TransitModel transitModel,
+    @Nullable StopConsolidationRepository repo,
+    GraphBuilderDataSources dataSources,
+    BuildConfig buildConfig
+  ) {
+    if (buildConfig.stopConsolidationFile == null) {
+      return null;
+    } else {
+      var file = dataSources
+        .baseDirectory()
+        .toPath()
+        .resolve(buildConfig.stopConsolidationFile)
```

Comment:
Not deal breaking, but the initial notice about excluded files includes the value for `stopConsolidationFile`, even if specified in `build-config.json`. People won't know that the file was recognized until the `StopConsolidationModule` actually runs.

Reply:
@t2gran has also asked this to be reworked so there might be changes coming up.